### PR TITLE
Mute expected error: BLOCK_BY_CONSENT

### LINKED ISSUE #15871: Error: BLOCK_BY_CONSENT firing too many times
The following error is causing logs to be too noisy: 
```
Error: BLOCK_BY_CONSENT
at getLayoutBox (https://raw.githubusercontent.com/ampproject/amphtml/1528247370025/src/service/resource.js:616)
```

If the error expected, we could consider muting it.

### PR DESCRIPTION
**Pull request by zhouyx**
Fix #15871

### CODE REVIEW

File: src/error.js [hunk 1]
```diff
@@ -550,7 +552,7 @@ export function resetAccumulatedErrorMessagesForTesting() {
  * @visibleForTesting
  */
 export function detectJsEngineFromStack() {
-  /** @constructor */
+  /** @class */
```
Comment (cathyxz) [inline src/error.js]: Why this change?

Comment (zhouyx) [reply src/error.js]: was trying to make linter happy. But type check doesn't like the change. I added `// eslint-disable-line` instead.

cc @rsimha

Comment (rsimha) [reply src/error.js]: If closure compiler prefers `@constructor` over `@class`, we can specify that preference here. There might be other places where we'd need to change the tag, but it should be a simple find replace, I'd imagine.

https://github.com/ampproject/amphtml/blob/cbf3929bd9e867f38c18c1abeddd1957a7c46e29/.eslintrc#L39-L43

Comment (rsimha) [reply src/error.js]: Correct, it does prefer `@constructor`: https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#constructor

Comment (zhouyx) [reply src/error.js]: Thanks for the investigation.

I've created #15941. Will have a separate PR to fix this.

**Comment by zhouyx** [comment]
PTAL : )

**Pull request merged**